### PR TITLE
mgr/cephadm: add autopep8 format changes

### PR DESCRIPTION
**Pull request by mgfritch**
introduced by: 7d8a3b60321

Signed-off-by: Michael Fritch <mfritch@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by mgfritch** [comment]
It's getting rather annoying working around these in a dev. environment ... can we get the additional changes committed that did not occur during #36348 ?

**Comment by mgfritch** [comment]
jenkins make check

**Comment by mgfritch** [comment]
jenkins test make check

**Comment by sebastian-philipp** [comment]
Hm. My thought was: 

1. do we really want to stop running QA runs for a day, plus force everyone to rebase their PRs, just for some PEP-8 changes? 
2. I don't want to review this monster PR. Most of the changes look innocent, but some changes introduce bugs or break type annotations. 

That's why I asked @votdev to skip those changes in his pr and since then sneaked in PEP8 PRs once every few days.

**Comment by mgfritch** [comment]
> Hm. My thought was:
> 
> 1. do we really want to stop running QA runs for a day, plus force everyone to rebase their PRs, just for some PEP-8 changes?
> 2. I don't want to review this monster PR. Most of the changes look innocent, but some changes introduce bugs or break type annotations.
> 
> That's why I asked @votdev to skip those changes in his pr and since then sneaked in PEP8 PRs once every few days.

but would it be better to get this done now?

I'm unsure how other devs are working around this, but my current workflow has been to commit locally, run tox, and then stash/drop the autopep8 changes ..

File: src/pybind/mgr/cephadm/inventory.py [hunk 1]
```diff
@@ -172,9 +173,9 @@ def __init__(self, mgr):
         self.osdspec_previews = {}     # type: Dict[str, List[Dict[str, Any]]]
         self.networks = {}             # type: Dict[str, Dict[str, List[str]]]
         self.last_device_update = {}   # type: Dict[str, datetime.datetime]
-        self.daemon_refresh_queue = [] # type: List[str]
-        self.device_refresh_queue = [] # type: List[str]
-        self.osdspec_previews_refresh_queue = [] # type: List[str]
+        self.daemon_refresh_queue = []  # type: List[str]
+        self.device_refresh_queue = []  # type: List[str]
+        self.osdspec_previews_refresh_queue = []  # type: List[str]
```
Comment (sebastian-philipp) [inline src/pybind/mgr/cephadm/inventory.py:178]: this change brakes the alignment.

File: src/pybind/mgr/cephadm/tests/test_scheduling.py [hunk 2]
```diff
@@ -161,17 +165,23 @@ def run_scheduler_test(results, mk_spec, get_hosts_func, get_daemons_func, key_e
...
+                             "Cannot place <ServiceSpec for service_name=mon> on 2: Unknown hosts")),
+    (k("1   123 * h"), error(OrchestratorValidationError,
+                             "Cannot place <ServiceSpec for service_name=mon> on 2, 3: Unknown hosts")),
     (k("1   *   * *"), exactly('1')),
     (k("12  1   * *"), exactly('1')),
     (k("12  12  1 *"), one_of('1', '2')),
     (k("12  12  * *"), exactly('1', '2')),
-    (k("12  123 * h"), error(OrchestratorValidationError, "Cannot place <ServiceSpec for service_name=mon> on 3: Unknown hosts")),
+    (k("12  123 * h"), error(OrchestratorValidationError,
+                             "Cannot place <ServiceSpec for service_name=mon> on 3: Unknown hosts")),
```
Comment (sebastian-philipp) [inline src/pybind/mgr/cephadm/tests/test_scheduling.py:184]: the scheduler test is already **really** hard to read. breaking the table here makes it even harder to grasp. 

Can we leave this as it was?

**Review by sebastian-philipp — CHANGES_REQUESTED** [review]

**Comment by mgfritch** [comment]
Duplicated by: #36793, #36794, #36792, #36817, #36818, #36816, #36815

**Pull request closed without merging**